Author: Sergey B Kirpichev <skirpichev@gmail.com>
Description: Don't fail if there is a backslash followed by a whitespaces in comments of the openntpd.conf (Closes: #435753).

diff -Naurp openntpd.orig/parse.y openntpd/parse.y
--- openntpd.orig/parse.y	2006-05-14 05:29:23.000000000 +0000
+++ openntpd/parse.y	2009-01-10 22:02:45.000000000 +0000
@@ -345,7 +345,7 @@ yylex(void)
 
 	yylval.lineno = lineno;
 	if (c == '#')
-		while ((c = lgetc(fin)) != '\n' && c != EOF)
+		while ((c = getc(fin)) != '\n' && c != EOF)
 			; /* nothing */
 
 	switch (c) {
